spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -584,433 +513,568 @@
-            expect(pending_enrollment.status_updated_at).to eq(Time.zone.now - 1.day)
-            expect(pending_enrollment.status_check_attempted_at).to eq(Time.zone.now)
-            expect(pending_enrollment.status_check_completed_at).to eq(Time.zone.now)
+        context 'when there is no status update' do
+          before(:each) do
+            stub_request_in_progress_proofing_results
           end
 
-          expect(pending_enrollment.profile.active).to eq(false)
-          expect(pending_enrollment.pending?).to be_truthy
+          it 'updates the timestamp but does not update the status or log a message' do
+            freeze_time do
+              pending_enrollment.update(
+                enrollment_established_at: Time.zone.now - 3.days,
+                status_check_attempted_at: Time.zone.now - 1.day,
+                status_updated_at: Time.zone.now - 1.day,
+              )
 
-          expect(job_analytics).not_to have_logged_event(
-            'GetUspsProofingResultsJob: Enrollment status updated',
-          )
+              job.perform(Time.zone.now)
+
+              pending_enrollment.reload
+              expect(pending_enrollment.enrollment_established_at).to eq(Time.zone.now - 3.days)
+              expect(pending_enrollment.status_updated_at).to eq(Time.zone.now - 1.day)
+              expect(pending_enrollment.status_check_attempted_at).to eq(Time.zone.now)
+              expect(pending_enrollment.status_check_completed_at).to eq(Time.zone.now)
+            end
+
+            expect(pending_enrollment.profile.active).to eq(false)
+            expect(pending_enrollment.pending?).to be_truthy
+
+            expect(job_analytics).not_to have_logged_event(
+              'GetUspsProofingResultsJob: Enrollment status updated',
+            )
+          end
         end
-      end
 
-      context 'when a timeout error occurs' do
-        before(:each) do
-          stub_request_proofing_results_with_timeout_error
+        context 'when a timeout error occurs' do
+          before(:each) do
+            stub_request_proofing_results_with_timeout_error
+          end
+
+          it_behaves_like(
+            'enrollment_encountering_an_error_that_has_a_nil_response',
+            error_type: Faraday::TimeoutError,
+          )
         end
 
-        it_behaves_like(
-          'enrollment_encountering_an_error_that_has_a_nil_response',
-          error_type: Faraday::TimeoutError,
-        )
+        context 'when a nil status error occurs' do
+          before(:each) do
+            stub_request_proofing_results_with_nil_status_error
+          end
+
+          it_behaves_like(
+            'enrollment_encountering_an_error_that_has_a_nil_response',
+            error_type: Faraday::NilStatusError,
+          )
+        end
       end
 
-      context 'when a nil status error occurs' do
-        before(:each) do
-          stub_request_proofing_results_with_nil_status_error
+      describe 'DAV enabled' do
```

Comment:
line 1029-1077 is the new addition to the spec. The rest of the tests in "describe 'ipp enabled'" were moved into another "describe" block (dav not enabled) which is why there are so many changes in this file